configs/islandora-HEAD.make

```diff
@@ -0,0 +1,95 @@
+; Run this from within the sites/default or sites/all directory, whichever you prefer:
```

Comment:
I was thinking in terms of multiple possible makefiles that might not grab identical lists of modules, like when the list of modules changes between two versions, or when people are using a custom makefile.